prime-router/src/test/resources/validation/marsotcelr/sample_3.hl7

```diff
@@ -0,0 +1,9 @@
+MSH|^~\&|MMTC.PROD^2.16.840.1.113883.3.8589.4.2.106.1^ISO|CAREEVOLUTION^00Z0000024^CLIA|AIMS.INTEGRATION.PRD^2.16.840.1.114222.4.3.15.1^ISO|AIMS.PLATFORM^2.16.840.1.114222.4.1.217446^ISO|20240403205305+0000||ORU^R01^ORU_R01|20240403205305_dba7572cc6334f1ea0744c5f235c823e|P|2.5.1|||NE|NE|||||PHLabReport-NoAck^ELR251R1_Rcvr_Prof^2.16.840.1.113883.9.11^ISO
```

Comment:
It doesn't look like this test file is being used

Reply:
I wanted to include it so that it could be used for onboarding receivers or as a sample file to explain stuff to senders. I am updating the name of this file to be more descriptive.

Reply:
Also using it in my test now.